### DISCUSSION TITLE
November update

### DISCUSSION THREAD

**Discussion by andykrd** [hldh214/lok_bot#79] [Q&A]
Hi guys!
there have been several updates to the game over the past months - please advise how the script works or not?
Bans arrive or not?

_Question to the developers_ - is there an update planned or is everything working properly and in general there is nothing more to improve?

**Comment by hldh214** [comment]
According to this issue we discussed several months ago https://github.com/hldh214/lok_bot/issues/75#issuecomment-1166175376

Since the game using encryption for their API, there is no way to continue maintaining this script, at least, that's what I think.

So, sorry to those who have high expectations for this project. But for now, this project is dead.
